xcat-inventory/xcclient/inventory/schema/1.0/osimage.yaml

```diff
@@ -14,14 +14,18 @@
       groups: "T{osimage.groups}"
     role: 
     - "T{osimage.profile}"
-    - "C:${{value=V{role}: True if str(value) in ('compute','service') else False}}"
+    - "C:${{value=V{role}: True if str(value) in ('','compute','service') else False}}"
     provision_mode: 
     - "T{osimage.provmethod}"
     - "C:${{value=V{provision_mode}: True if str(value) in ('install','netboot','statelite') else False}}"
     package_selection:
-      pkglist: "T{linuximage.pkglist}"
+      pkglist: 
+      - "T{linuximage.pkglist}"
+      - "F:${{flist=V{package_selection.pkglist}.split(','): [x for x in flist if not vutil.underpath(x,'/opt/xcat/share/xcat/') ] }}"
```

Comment:
Let's use a more general method there get the all files need to be backup (cover INCLUDE ),  `underpath` is just one condition.

Reply:
yes, will create a feature request on this. Will enhance this in another PR